tests/TransportFunctionalTest.php

```diff
@@ -52,6 +52,13 @@ protected function setUp(): void
         $transport = $container->get('messenger.transport.test_phpamqplib');
         assert($transport instanceof AmqpTransport);
 
+        // "purge" the queues
+        foreach ($transport->getConnection()->getQueueNames() as $queue) {
+            foreach ($transport->getConnection()->get($queue) as $message) {
+                $message->ack();
```

Comment:
not sure if there's a better way to empty the queues, I personally use the RMQ admin REST API for this, but that's not readily available here.